DQMOffline/Trigger/src/HLTMuonRefMethod.cc

```diff
@@ -0,0 +1,228 @@
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "DQMServices/Core/interface/DQMEDHarvester.h"
+#include "FWCore/Framework/interface/Event.h"
+
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include <set>
+#include <string>
+#include <vector>
+#include <TPRegexp.h>
+#include <cmath>
+#include <climits>
+#include <boost/tokenizer.hpp>
+
+
+#include <TH1.h>
+#include <RVersion.h>
+#if ROOT_VERSION_CODE >= ROOT_VERSION(5,27,0)
+#include <TEfficiency.h>
+#else
+#include <TGraphAsymmErrors.h>
+#endif
+
+
+TPRegexp metacharacters("[\\^\\$\\.\\*\\+\\?\\|\\(\\)\\{\\}\\[\\]]");
```

Comment:
no global variables (but in any case, use standard string parsing tools, not root ones)